spec/swagger.json

```diff
@@ -0,0 +1,51 @@
+{
+  "swagger": "2.0",
```

Comment:
Why not use OpenAPI 3.x here and not use dapperdox? We are trying to get rid of DapperDox and OpenAPI 3 was announced in 2017.